CalendarApp/NUnitTestProject/BookingServiceTests.UpdateTest.cs

```diff
@@ -0,0 +1,83 @@
+﻿using NUnit.Framework;
+using Google.Apis.Calendar.v3.Data;
+using System.Linq;
+using BookingRoom.Helpers;
+
+namespace NUnitTestProject
+{
+    [TestFixture]
+    public partial class BookingServiceTests 
+    {
+        [Test]
+        public void UpdateEvent_WithValidInputData_UpdatesEventCorrectly()
+        {
+            var calendarId = AppSettingsHelper.GetAppSetting(AppSetingsConst.TestCalendar);
+
+            var beforeUpdateEventsList = _calendarService.Events.List(calendarId).Execute().Items;
+
+            var testEvent = beforeUpdateEventsList[0];
+
+            var currentSummary = testEvent.Summary; //remember current state
+            var currentDesctiption = testEvent.Description;
+            testEvent.Summary = $"Changed{currentSummary}"; //chenage event
+            testEvent.Description = $"Chenged{currentDesctiption}";
+
+            _meetingBooking.UpdateEvent(testEvent, calendarId, testEvent.Id);
+
+            var afterUpdateList = _calendarService.Events.List(calendarId).Execute().Items;
+
+            var testedAfterUpdateEvent = afterUpdateList.FirstOrDefault(ev => ev.Id == testEvent.Id);
+           
+            Assert.AreEqual(testEvent.Summary, testedAfterUpdateEvent.Summary); //we cant just Assert.AreEqual(two this object) couse of "updated" time field will be different anyway
+            Assert.AreEqual(testEvent.Description, testedAfterUpdateEvent.Description);
+
+            testedAfterUpdateEvent.Summary = currentSummary;//push event back to past state
+            testedAfterUpdateEvent.Description = currentDesctiption;
+
+            _meetingBooking.UpdateEvent(testedAfterUpdateEvent, calendarId, testedAfterUpdateEvent.Id);
+
+            var checkBackEvent = beforeUpdateEventsList.FirstOrDefault(ev => ev.Id == testEvent.Id);
+            
+            Assert.AreEqual(checkBackEvent.Summary, testEvent.Summary);
+            Assert.AreEqual(checkBackEvent.Description, testEvent.Description);
+        }
```

Comment:
I'd suggest to add and delete event instead of updating existing event back and forth